app/views/events/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<h1 class="title title-event is-4 navbar-item is-capitalized">Here is a list of our events.</h1>
+
+<div class="event-tables tables">
+  <table>
+    <thead>
+      <tr>
+        <th>
+          <%= fa_icon "calendar" %>
+          <span>Upcoming Events</span>
+        </th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @upcoming_events.each do |event| %>
+        <tr class="box box-radius">
+          <td class="column">Event: <%= link_to event.title, event %></td>
+          <td class="column"><div class="description">Description: <%= event.description %><div></td>
+          <td class="column">Date: <%= event.date.to_formatted_s(:long_ordinal)  %></td>
+          <td class="column">Creator: <%= event.creator.name %></td>
+          <td class="column"><%= link_to 'Delete', event, method: :delete, class:'button is-primary has-text-white'  %></td>
```

Comment:
- [ ] Any user can delete any event even if he/she is not the creator of the event, let's add a check to render delete button only if the current user is the same as the event creator. (Don't forget to wrap this logic in events helper module `app/helpers/`).